treeherder/push_health/utils.py

```diff
@@ -44,8 +49,7 @@ def clean_test(test_name):
 
     # Now that we don't bail on a blank test_name, these filters
     # may sometimes apply.
-    if clean_name in [None,
-                      'Last test finished',
+    if clean_name in ['Last test finished',
```

Comment:
None actually never gets here now because we look at different fields depending on the `action`.